src/PPN-microbench/asm/cpu_frequency_arm_64.s

```diff
@@ -0,0 +1,32 @@
+.global cpu_frequency_arm_64
+
+cpu_frequency_arm_64:
+    mov x0, x0 
+    mov x1, #0 
+    lsl x0, x0, #4 
+
+1:
```

Comment:
Same here!
```suggestion
.1:
```
Don't forget to update the jump at the end of the loops, and the file name with a capital S.